notebooks/image_models/solutions/conditional_gan.ipynb

```diff
@@ -0,0 +1,1190 @@
+{
```

Comment:
<p>Same comment as before, this is too much code for one cell. Consider splitting this cell up.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>The entire code in this cell is one class actually. I will try to reduce the size, but is it ok to put it in one cell? Or do you think it is better to apply different approach?</p>

Reply:
<p>Thanks for your comment.</p><p>Since this is a single class and I believe this is the minimum setup, I added a comprehensive explanations in the markdown cell above.</p><p>Hope it works for you!</p>